containers/src/Utils/NoThunks.hs

```diff
@@ -0,0 +1,15 @@
+module Utils.NoThunks (whnfHasNoThunks) where
```

Comment:
I think this file should be in `containers-tests/tests/Utils/`.

Reply:
Yes, it's ought to be there.

Reply:
I put it under `containers-tests/src` which is the only directory in `hs-source-dirs` of `containers-test:lib:containers-test`.

Reply:
`containers-tests/src` is a symlink to `containers/src`, i.e. not a good place for test things. Why not put it next to `Utils.IsUnit`?!

Reply:
I just realised this, quite surprising. I'll put it next to `Utils.IsUnit`.

Reply:
Done.